recipes/openassetio/all/conanfile.py

```diff
@@ -0,0 +1,172 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class PackageConan(ConanFile):
+    name = "openassetio"
+    description = "An open-source interoperability standard for tools and content management systems used in media production."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/project/package"
+    topics = ("asset-pipeline", "vfx", "cg", "assetmanager", "vfx-pipeline")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "without_python": [True, False],
+        "python_version": ["3.7.12", "3.8.12", "3.9.7", "3.10.0"]
```

Comment:
[Question] I'd expected to see 3.7, 3.9, 3.10 and 3.11 here. Is this list something other than our supported python set?